### ISSUE TITLE
Usersnap Feedback - Component Field BUG - I should not be able to select the current case as a [...]

### ISSUE THREAD

**Issue by scottofletcher** [participedia/frontend#846]
_From @davidascher on June 18, 2018 23:14_

**Sender**: scottowenfletch@gmail.com
**Comment**: Component Field BUG - I should not be able to select the current case as a component or parent. 
[Open #211 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-ed971123/211)

<a href='https://usersnappublic.s3.amazonaws.com/2018-06-19/01-14/3625dc1e-b12b-40ab-a418-3f2096810f4a.png'>![Usersnap Feedback - Component Field BUG - I should not be able to select the current case as a (...)](https://usersnappublic.s3.amazonaws.com/2018-06-19/01-14/fa89c504-6601-4a67-8e1a-2e4a3f12a6da.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2018-06-19/01-14/3625dc1e-b12b-40ab-a418-3f2096810f4a.png'>Download original image</a>

**Browser**: Firefox 60 (Windows 10)
**Referer**: [https://participedia.xyz/case/5317/edit](https://participedia.xyz/case/5317/edit)
**Screen size**: 1920 x 1080 **Browser size**: 1920 x 876

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

_Copied from original issue: participedia/usersnaps#198_

**Comment by jesicarson** [comment]
Good point. Not a bug, but could be coded: List all the cases in the database as possible components/ parents EXCEPT FOR the case currently being edited (because it can't be/ shouldn't be listed as a component or parent of itself). OR, show an error message when a user attempts to publish a case with components and parents of itself. Whatever is easier.

At any rate, I just tested it and luckily nothing breaks when you do this. It just lists the current case under components in the data sidebar, which is weird, but hopefully not many people will do it. I would maybe switch this to P2 for this reason - unlikely to occur because hopefully most people will just not do it.
